firebase-firestore/src/main/java/com/google/firebase/firestore/local/MemoryRemoteDocumentCache.java

```
@@ -81,6 +85,35 @@ public MutableDocument get(DocumentKey key) {
     return result;
   }
 
+  @Override
+  public Map<DocumentKey, MutableDocument> getAll(
+      String collectionGroup, IndexOffset offset, int limit) {
+    // Note: This method is pretty inefficient, but t is not called since the method is only used
```

t -> it

---

Also, if this is the case then maybe just throw an exception here?